ChangeLog.md

```diff
@@ -205,6 +207,11 @@ Notable changes to the FreeOrion project will be documented in this file.
 - Fixed issues with AIs replaying their turns when loading a game.
 - Fixed combat log layout / scrollbar issues.
 - Fixed issues with compressed XML saves containing invalid XML.
+- Fixed issue where fleets which had never been moved since their creation were unable to enforce blockades.
+- Fixed segfault scrolling through tabs.
+- Use correct auto aggression icon for new fleet aggression button tooltip.
+- Fixed missing combat sensor ghosts.
```

Comment:
and this?

Reply:
IIRC, there has been a long-standing issue with visibility/sensor ghosts after combats in certain situations, which I thought @Dilvish-fo finally got around to fix.

I'm not 100% sure though - @Dilvish-fo? That fix is one of yours, has that bug been around pre 0.4.7?